lib/plugins/aws/info/display.js

```diff
@@ -1,67 +1,34 @@
 'use strict';
 
-const chalk = require('chalk');
 const filesize = require('../../../utils/filesize');
-const { legacy, isVerboseMode, style } = require('@serverless/utils/log');
+const { isVerboseMode, style } = require('@serverless/utils/log');
 
 module.exports = {
   displayServiceInfo() {
-    const info = this.gatheredData.info;
-
-    let message = '';
-    message += `${chalk.yellow.underline('Service Information')}\n`;
-    message += `${chalk.yellow('service:')} ${info.service}\n`;
-    message += `${chalk.yellow('stage:')} ${info.stage}\n`;
-    message += `${chalk.yellow('region:')} ${info.region}\n`;
-    message += `${chalk.yellow('stack:')} ${info.stack}\n`;
-    message += `${chalk.yellow('resources:')} ${info.resourceCount}`;
-
-    if (info.resourceCount >= 450) {
-      message += `\n${chalk.red('WARNING:')}\n`;
-      message += `  You have ${info.resourceCount} resources in your service.\n`;
-      message += '  CloudFormation has a hard limit of 500 resources in a service.\n';
-      message += '  For advice on avoiding this limit, check out this link: http://bit.ly/2IiYB38.';
-    }
-
-    legacy.consoleLog(message);
     if (this.serverless.processedInput.commands.join(' ') === 'info') {
       this.serverless.serviceOutputs.set('service', this.serverless.service.service);
       this.serverless.serviceOutputs.set('stage', this.provider.getStage());
       this.serverless.serviceOutputs.set('region', this.provider.getRegion());
       this.serverless.serviceOutputs.set('stack', this.provider.naming.getStackName());
     }
-
-    return message;
   },
 
   displayApiKeys() {
-    const conceal = this.options.conceal;
+    // TODO: support conceal option
```

Comment:
I've noticed that when changing how we construct the outputs, the `conceal` option is not respected - we should support it, right?

Reply:
Good question.

I think my reasoning when configuring legacy logs, was that this is a legacy option (as it was not documented in CLI _help_), which there's no valid reason to support with modern logs, and as we won't use it in modern logs it's safe to ignore.

However, as I check in telemetry stats, this option has some usage. e.g. during last month 145 of users, have used it (0.45% of total), so we may revisit it.

Either we should:
- Provide proper support for it, by listing it in command options, and adapting our modern logs output to it
- Drop it (drop any documentation on it in v3, as I see it's documented on two markdown pages)

I propose to raise it during the next meeting (/cc @mnapoli)

Reply:
Thanks for response - let's raise it during next meeting 👍

Reply:
and just to make sure I understand, this is a `--conceal` CLI option? It's available in all commands, or just a specific one?

Reply:
it's available for `deploy` and `info` commands and yes, it's a CLI option